docs/tutorials/hello-tye/00_run_locally.md

```diff
@@ -65,24 +65,9 @@ This tutorial will demonstrate how to use [`tye run`](/docs/reference/commandlin
 
 Now that we have two applications running, let's make them communicate. By default, `tye` enables service discovery by injecting environment variables with a specific naming convention. For more information on, see [service discovery](/docs/reference/service_discovery.md).
 
-1. Add a NuGet.config to add the `dotnet-core` package source.
-
-    Paste the following into `nuget.config` in the `microservice/` directory.
-
-    ```xml
-    <?xml version="1.0" encoding="utf-8"?>
-    <configuration>
-        <packageSources>
-            <clear />
-            <add key="dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
```

Comment:
Is this information available as part of using our nightly builds?

Reply:
yes. This is still part of the getting started section.